pkg/services/ngalert/api/lotex_ruler_test.go

```diff
@@ -24,65 +24,60 @@ func TestLotexRuler_ValidateAndGetPrefix(t *testing.T) {
 		expected        string
 		err             error
 	}{
-		{
-			name:        "with an invalid datasource ID",
```

Comment:
Can we change this test to assert if an empty string returns an error instead? (if we don't have a similar test yet)